### DISCUSSION TITLE
PWA docs/website

### DISCUSSION THREAD

**Discussion by sukiejosh** [saadeghi/daisyui#1851] [General]
Hi, would you mind turning daisyui.com to PWA, it would be very helpful offline ,thanks for the components

**Comment by saadeghi** [comment]
Sure. But until then, you can clone this repo and build the docs on local to use it offline

**Comment by sukiejosh** [comment]
Ok thanks

**Comment by oekazuma** [comment]
What about using `vite-plugin-pwa`?

docs: https://vite-plugin-pwa.netlify.app/frameworks/sveltekit.html

**Comment by saadeghi** [comment]
@oekazuma looks good. I will try that

**Comment by Korak-997** [comment]
Is this ISSUE still open ? i may be able to work on it !

**Comment by saadeghi** [comment]
@Korak-997 
Yes.  
Any contribution is appreciated of course.

I tried for a PWA in SvelteKit but that didn't work and it was low priority for me to spend more time for.

**Comment by Korak-997** [comment]
@saadeghi 
is there any Contributions guide which i have to follow ? Or simply clone the project and make a PR ?

**Comment by saadeghi** [comment]
@Korak-997 Here: https://github.com/saadeghi/daisyui/blob/master/.github/CONTRIBUTING.md

Let me know if you have any questions.

**Comment by Korak-997** [comment]
@saadeghi looking great thanks :+1: 

If is possible you can assign the Issue to me , and i will look through it but i can not promise to fix it fast. Because of my work and studies i do not have so much free time.

**Comment by saadeghi** [comment]
Sure. There's no time pressure. Don't worry.

**Comment by 0b1kn00b** [comment]
Trying to build getting: `"LEMONSQUEEZY_API_KEY" is not exported by "virtual:$env/static/private", imported by "src/routes/store/+layout.server.js".`
